src/internal/testutil/identity.go

```diff
@@ -92,9 +92,7 @@ func ConfigureOIDCProvider(t *testing.T) error {
 }
 
 // DoOAuthExchange does the OAuth dance to log in to the mock provider, given a login URL
-func DoOAuthExchange(t testing.TB, loginURL string) {
-	testClient := GetUnauthenticatedPachClient(t)
-
+func DoOAuthExchange(t testing.TB, pachClient, enterpriseClient *client.APIClient, loginURL string) {
```

Comment:
This used to assume dex and pachd were all at one address - extend it to support a separate enterprise server context with different ports and/or a different address.